packages/faustwp-cli/utils/marshallTelemetryData.ts

```diff
@@ -0,0 +1,100 @@
+import fs from 'fs';
+import { getCliArgs } from './getCliArgs.js';
+import { WPTelemetryResponseData } from './requestWPTelemetryData.js';
+
+export interface TelemetryData {
+  wordpress: {
+    faustwp_settings: {
+      has_frontend_uri?: boolean;
+      redirects_enabled?: boolean;
+      rewrites_enabled?: boolean;
+      themes_disabled?: boolean;
+      image_source_replacement_enabled?: boolean;
+    };
+    faustwp_version?: string;
+    is_wpe?: boolean;
+    multisite?: boolean;
+    php_version?: string;
+    wp_version?: string;
+  };
+  node: {
+    faust_version?: string;
```

Comment:
We may want to rename this to `faust_core_version`

```suggestion
    faust_core_version?: string;
```